src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,12 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import './TodoInfo.scss';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : null}`}>
+    <h2 className="TodoInfo__title">
+      {todo.title}
+    </h2>
+
+    <UserInfo user={todo.user} />
+  </article>
```

Comment:
UserInfo takes a user and renders the details (**use it only if todo.user exists**)

need to render UserInfo component by condition (when todo.user exists)